### ISSUE TITLE
analyzer should have an unused ignore warning

### ISSUE THREAD

**Issue by DetachHead** [dart-lang/sdk#49834]
```dart
void main() {
  // ignore: unused_local_variable
  var foo = 1;
  print(foo);
}
```
it would be cool if the analyzer warned that this `ignore` comment is unused.

**Comment by bwilkerson** [comment]
Duplicate of https://github.com/dart-lang/sdk/issues/35234.

Summary: we have a working implementation, but there's a lot of code that needs to be cleaned up before we can land it.
